### DISCUSSION TITLE
Token Curated Registry

### DISCUSSION THREAD

**Discussion by stephen-rowan** [SecretDecks/Documentation#7] [Ideas]
## How to prototype a Token Curated Registry ?

Hello everyone,

I noticed the "[Token Design: Kickoff Document](https://github.com/SecretDecks/Documentation/blob/main/TokenDesign-Kickoff-Doc.md)"  mentioned that SecretDecks could "curate artworks in a decentralized and autonomus manner (via Token Curated Registry [TCR])".

A prototype for this could be built based upon the [Cardano Foundation Metadata Registry](https://forum.cardano.org/t/introducing-the-cardano-token-registry-for-on-chain-identifiers/57594) method.

SecretDecks could start with metadata definitions of curation descriptions. These could either be freshly invented or based on some standard (here is an article on [Image Metadata Standards](https://acadi.wordpress.com/image-curation/metadata-standards/) )

Then these metadata definitions would be submitted to a SecretDecks Metadata Registry repo for review (we could automate this process using GitHub Actions) where they would form the SecretDecks art description corpus.

The corpus of SecretDecks metadata would then become available to autonomus artists either through an API or as a standard to be applied in a SecretDecks smart contract.

The activity of curation could be tokenised by transacting art criticism/description on an art object (through selection from the corpus). In short an autonomus art critic may submit metadata to the corpus and/or apply metadata to an art object. SecretDecks would charge a nominal fee to support its platform.

**Comment by DanM3rcurius** [comment]
genius. I'll write a task for me to dig into Metadata Standards!

**Comment by DanM3rcurius** [comment]
@bUxEE & @tamagit80 What do you think?
